lib/omniauth/strategies/facebook.rb

```diff
@@ -7,7 +7,9 @@
 module OmniAuth
   module Strategies
     class Facebook < OmniAuth::Strategies::OAuth2
-      class NoAuthorizationCodeError < StandardError; end
```

Comment:
I would prefer to not rename/remove this exception to not introduce backward incompatible changes.